### DISCUSSION TITLE
[p93 예제 2-30.] this.userId = userId;

### DISCUSSION THREAD

**Discussion by codingwanee** [fsoftwareengineer/todo-application#14] [질문과 답변] [answered]
책 보고 공부에 도움 많이 받고 있습니다. 작은 편집오류 발견해서 글씁니다.
내용상 오류는 아니지만 예제의 this.uerId = 다음 줄바꿈이 두 번 들어간 뒤 userId;가 나옵니다.
소스를 그대로 따라치며 공부하는 초보가 보기에 혼돈이 있을 수 있을 것 같습니다.

**Comment by fsoftwareengineer** [comment, marked as answer]
> 책 보고 공부에 도움 많이 받고 있습니다. 작은 편집오류 발견해서 글씁니다.
> 
> 내용상 오류는 아니지만 예제의 this.uerId = 다음 줄바꿈이 두 번 들어간 뒤 userId;가 나옵니다.
> 
> 소스를 그대로 따라치며 공부하는 초보가 보기에 혼돈이 있을 수 있을 것 같습니다.

감사합니다. 수정하도록 하겠습니다.
